server/src/test/java/org/elasticsearch/common/logging/DeprecationLoggerTests.java

```diff
@@ -197,19 +196,6 @@ public void testCanRemoveThreadContext() throws IOException {
         }
     }
 
-    public void testIgnoresClosedThreadContext() throws IOException {
```

Comment:
removing as we don't expect closed contexts to be present in DeprecatedLogger. It should be removed before closing